src/Widgets/Button/Button.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export const Button = ({ children }) =>
+  <StyledButton>{ children }</StyledButton>
+
+const StyledButton = styled.button`
+  align-self: center;
+  padding: 20px 20px 20px 20px;
+  color: white;
+  font-size: 18px;
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  text-align: center;
+  background: #2ecc71;
+  border-radius: 10px;
+  transition: all 0.1s ease-in-out;
```

Comment:
```suggestion
  transition: all 0.1s ease-in-out;
```

Never use transition: all.